extensions/wikia/Email/EmailController.class.php

```diff
@@ -396,6 +406,80 @@ protected function getSocialMessages() {
 		];
 	}
 
+	private function generateMobileApplicationBadges() {
+		$mobileApplicationsLinks = $this->generateMobileApplicationLinks();
+		$hasMobileApplicationBadges = count($mobileApplicationsLinks) > 0;
```

Comment:
Trivial: make sure to have spacing inside of function/method calls. It's part of our PHP guidelines.